x/profiles/keeper/keeper_dtag_transfers.go

```diff
@@ -25,6 +25,8 @@ func (k Keeper) SaveDTagTransferRequest(ctx sdk.Context, request types.DTagTrans
 
 	requests = types.NewDTagTransferRequests(append(requests.Requests, request))
 	store.Set(key, k.cdc.MustMarshalBinaryBare(&requests))
+
+	k.Logger(ctx).Info("dTag transfer request", "receiver", request.Receiver, "from", request.Sender)
```

Comment:
I think it's better to use the conventional `DTag` here

```suggestion
	k.Logger(ctx).Info("DTag transfer request", "receiver", request.Receiver, "from", request.Sender)
```